blockonomics.php

```diff
@@ -708,4 +755,48 @@ public function generatenewCallback()
                 $secret
         );
     }
+
+    public function getTempApiKey($callback_url)
+    {
+        $url = Configuration::get(
+            'BLOCKONOMICS_TEMP_API_KEY_URL'
+        );
+        $body = json_encode(array('callback' => $callback_url));
+        $response = $this->doCurlCall($url, $body);
+        $responseObj = $response->data;
+        $responseObj->{'response_code'} = $response->response_code;
+        return $responseObj;
+    }
+
+    public function makeWithdraw()
+    {
+        $api_key = $this->getApiKey();
+        $temp_api_key = Configuration::get(
+            'BLOCKONOMICS_TEMP_API_KEY'
+        );
+        if (!$api_key || !$temp_api_key || $temp_api_key == $api_key) {
+            return null;
+        }
+        $temp_withdraw_amount = Configuration::get(
+            'BLOCKONOMICS_TEMP_WITHDRAW_AMOUNT'
+        );
+        if ($temp_withdraw_amount > 0) {
+            $url = Configuration::get(
+                'BLOCKONOMICS_TEMP_WITHDRAW_URL'
+            ) .'?tempkey='.$temp_api_key;
+            $response = $this->doCurlCall($url);
```

Comment:
This does not work. `doCurlCall()` is GET but withdraw request must be POST

Reply:
Fixed with commit 3124e72